native_client/client.cc

```diff
@@ -221,6 +221,7 @@ GetAudioBuffer(const char* path)
 
   unsigned int sample_rate;
   fseek(wave, 24, SEEK_SET); rv = fread(&sample_rate, 4, 1, wave);
+  res.sample_rate = (int)sample_rate;
```

Comment:
we assert on the sample rate below, you likely want to check / remove that as well

Reply:
The only problem here is that the field is never initialized so we pass a garbage value to the API when we call DS_SpeechToText. I still want to keep the assert.